catroidTest/src/org/catrobat/catroid/uitest/content/brick/PlaceAtBrickTest.java

```diff
@@ -94,6 +95,26 @@ public void testPlaceAtBrick() throws InterruptedException {
 				R.id.brick_place_at_edit_text_y, yPosition, Brick.BrickField.Y_POSITION, placeAtBrick);
 	}
 
+	public void testBehaviorOfUndoAndRedoButton(){
+		solo.clickOnText("105");
+		solo.clickOnButton("5");
+		solo.waitForView(solo.getView(R.id.menu_undo));
```

Comment:
Hmm, I'm not sure if waitForView works that way. waitForView only waits until the view is visible on the screen. Since it is already visible on the keyboard, the method will return immediately.

A more suited method for the checking would be:
```java
solo.waitForCondition(new Condition() {
    @Override
    public boolean isSatisfied() {
        return undo.isEnabled();
    }
}, 50);
```
A oneline lambda version of this code would be neat. :wink: but for now a helper method is surely sufficient.